src/app/body/task-details/upload-files/upload-files.component.html

```diff
@@ -36,7 +36,7 @@ <h6 class="col-md-11 col-10 pt-2 mb-0">Upload Files</h6>
     <form>
         <div class="form-group">
             <label class="btn btn-outline-secondary btn-lg btn-block p-2" for="exampleFormControlFile1">
-              <h5><span class="material-icons">cloud_upload</span> Click to Upload Document</h5>
+              <h5><span class="material-icons" style="position: relative; top: 7px; right: 2px;">cloud_upload</span> Click to Upload Document</h5>
```

Comment:
The properties of icon is static, fix is to bootstrap standards. Use mat-icon and padding and a tag to fix this.

Reply:
find a better approach

Reply:
done